src/Microsoft.ML.Data/DataLoadSave/Text/TextLoader.cs

```diff
@@ -1460,7 +1460,8 @@ void ICanSaveModel.Save(ModelSaveContext ctx)
                 var memberInfo = memberInfos[index];
                 var mappingAttr = memberInfo.GetCustomAttribute<LoadColumnAttribute>();
 
-                host.Assert(mappingAttr != null, $"Field or property {memberInfo.Name} is missing the {nameof(LoadColumnAttribute)} attribute");
+                if (mappingAttr == null)
+                    throw host.Except($"{(memberInfo is FieldInfo ? "Field" : "Property")} '{memberInfo.Name}' is missing the {nameof(LoadColumnAttribute)} attribute");
```

Comment:
>memberInfo.Name [](start = 93, length = 15)

Just a note, while we often use `'` to indicate quoting of things that can contain potentially confusing items like spaces, since C# identifiers cannot contain spaces I view their presence as less essential than I would in, say, a column name or some other such entity. (Not a big deal at all, just registering a random thought since I am half allergic to letting PRs pass with no comments whatsoever. :D )